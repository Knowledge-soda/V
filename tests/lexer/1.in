f(x, 12, g(x, -5))
