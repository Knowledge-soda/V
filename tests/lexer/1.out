NAME(f) BEGIN NAME(x) SEP NUM(12) SEP NAME(g) BEGIN NAME(x) SEP NUM(-5) END END EOL
